test/resources/qaoa_entry_point.py

```diff
@@ -44,7 +44,7 @@ def record_test_metrics(metric, start_time, interface):
             'Unit': 'Seconds',
             'Value': time.time() - start_time
         }],
-        Namespace='braket-container-metrics'
+        Namespace='/aws/braket'
```

Comment:
Can you change the type to "braket_container_tests" so we know what the metrics relate to?

Reply:
Have you verified this got published to the right place?

Reply:
<img width="1466" alt="Screenshot 2023-12-01 at 12 06 45 PM" src="https://github.com/amazon-braket/amazon-braket-containers/assets/20567262/999444c3-d8b3-4e62-bed0-6a213ce4acdc">
Working as expected!